test/ui/shared/test-url-util.js

```diff
@@ -0,0 +1,61 @@
+/* global describe, it */
```

Comment:
We could probably have a custom .eslintrc to just say this in one place. We probably don't need to do that in this PR though